http-client/src/main/java/com/facebook/airlift/http/client/thrift/ThriftRequestUtils.java

```diff
@@ -25,10 +27,10 @@
 
 public class ThriftRequestUtils
 {
-    private static final MediaType THRIFT_TYPE = MediaType.create("application", "x-thrift");
-    private static final String TYPE_BINARY = THRIFT_TYPE.withParameter("t", Protocol.BINARY.name()).toString().toLowerCase();
-    private static final String TYPE_COMPACT = THRIFT_TYPE.withParameter("t", Protocol.COMPACT.name()).toString().toLowerCase();
-    private static final String TYPE_FBCOMPACT = THRIFT_TYPE.withParameter("t", Protocol.FB_COMPACT.name()).toString().toLowerCase();
+    public static final String TYPE_BINARY = "application/x-thrift+binary";
```

Comment:
If it's easy, I'd create an enum for this.

Reply:
`@Consumes` and `@Produces` annotations takes in a string. With an enum we will need to have an additional `.name()` invocation.
Also the `javax.ws.rs.core.MediaType` class which I have used as a reference uses strings as the mime types.